web/index.html

```diff
@@ -136,5 +136,6 @@
       }
     </script>
     <script src="assets/assets/js/package/olm.js"></script>
+    <script src="https://unpkg.com/pica/dist/pica.min.js" ></script>
```

Comment:
What is this

Reply:
https://pub.dev/packages/flutter_image_compress#web